### PR TITLE
[TW-1483] Live Insights edits

### PR DESCRIPTION
**Pull request by nerminamiller-postman**
Fixes email formatting, updates the Live Insights screenshot and adds small copyedits

- [About Live Insights Early Access](https://tw-1483-live-insights-edits.learning.postman-beta.com/docs/live-insights/live-insights-early-access/)
- [Live Insights overview](https://tw-1483-live-insights-edits.learning.postman-beta.com/docs/live-insights/live-insights-overview/)
- [Get started with Live Insights](https://tw-1483-live-insights-edits.learning.postman-beta.com/docs/live-insights/live-insights-gs/)

### CODE REVIEW

File: src/pages/docs/live-insights/live-insights-early-access.md [hunk 1]
```diff
@@ -16,29 +16,43 @@ contextual_links:
...
+*	Consolidated, per-endpoint metrics and insights
+*	Error pattern analysis
+*	More context about errors, enabling easier troubleshooting
+*	More automated insights
 
 The Live Insights team is hoping you will try out the new features and send your feedback to help Postman continue tailoring the product to your needs.
 
+## Documentation
+
+Just like Live Insights, this documentation set is in alpha stage, and therefore not listed in the left-side navigation. Use the links bellow to jump to the next topic of interest. For easier navigation, you'll find the list of other topics in this set at the bottom of each page.
```
Comment (jkonrath-postman) [inline src/pages/docs/live-insights/live-insights-early-access.md:39]:
```suggestion
Just like Live Insights, this documentation set is in alpha stage. Use the links bellow to jump to the next topic of interest. For easier navigation, you'll find the list of other topics in this set at the bottom of each page.
```

File: src/pages/docs/live-insights/live-insights-early-access.md [hunk 2]
```diff
@@ -16,29 +16,43 @@ contextual_links:
     url: "https://www.postman.com/product/live-insights/"
 ---
 
-The Live Insights team at Postman is working toward the open launch of Live Insights. Today, the alpha launch focuses on the Live Collections Agent (LCA), which passively watches your API traffic to automatically populate a Postman Collection with API endpoints. Within 15 minutes of installing the Live Collections Agent in staging or production, you’ll start to see endpoints in your collection. The Live Collections Agent will keep these endpoints up to date based on new observed traffic.
+The rise of SaaS and APIs means that production systems are more complex than ever before. The Postman vision for Live Insights is to make it faster and easier for any developer to understand their production behavior. By passively watching your API traffic, Live Insights provides API-centric data about what's going wrong with a system and where to start addressing the issue.
 
-API discovery is just the beginning of what Postman plans to offer. Akita users are familiar with the ability to explore API models based on error, latency, and volume.
+Advantages of using Live Insights:
```
Comment (nerminamiller-postman) [inline src/pages/docs/live-insights/live-insights-early-access.md:21]:
```suggestion
Advantages of using _Postman Live Insights_:
```

File: src/pages/docs/live-insights/live-insights-gs.md [hunk 3]
```diff
@@ -16,9 +16,9 @@ contextual_links:
     url: "https://www.postman.com/product/live-insights/"
 ---
 
-To get started using Live Insights, ensure you have a team set up in Postman. If you don’t already have a team in Postman, see [Collaboration overview](/docs/collaborating-in-postman/working-with-your-team/collaboration-overview/#creating-a-team) for instructions on how to set it up. Postman Free provides team setup with up to three users. Once your team is set up, contact [Live Insights Alpha team](live.insights.alpha@postman.com) to get access to the Live Insights alpha in Postman.
+To get started using Live Insights, ensure you have a team workspace set up in Postman. If you don’t already have a team in Postman, see [Collaboration overview](/docs/collaborating-in-postman/working-with-your-team/collaboration-overview/#creating-a-team) for instructions on how to set it up. Postman Free provides team setup with up to three users. Once your team workspace is set up, contact [Live Insights Alpha team](mailto:live.insights.alpha@postman.com) to get access to the Live Insights alpha in Postman.
```
Comment (nerminamiller-postman) [inline src/pages/docs/live-insights/live-insights-gs.md:19]:
```suggestion
To get started using Live Insights, ensure you have a team workspace set up in Postman. If you don’t already have a team in Postman, see [Collaboration overview](/docs/collaborating-in-postman/working-with-your-team/collaboration-overview/#creating-a-team) for instructions on how to set it up. The Postman Free plan provides team setup with up to three users. Once your team workspace is set up, contact [Live Insights Alpha team](mailto:live.insights.alpha@postman.com) to get access to the Live Insights alpha in Postman.
```

**Review by nerminamiller-postman — COMMENTED** [review]
Applies Jon’s feedback

**Pull request merged**